server/public/shared/mlog/tlog.go

```diff
@@ -33,12 +34,18 @@ func AddWriterTarget(logger *Logger, w io.Writer, useJSON bool, levels ...Level)
 }
 
 // CreateConsoleTestLogger creates a logger for unit tests. Log records are output to `os.Stdout`.
-// Logs can also be mirrored to the optional `io.Writer`.
-func CreateConsoleTestLogger(useJSON bool, level Level) *Logger {
-	logger, _ := NewLogger()
+// All log messanges with level trace or lower are logged.
+// The returned logger get Shutdown() when the tests completes. The caller should not shut it down.
+func CreateConsoleTestLogger(tb testing.TB, useJSON bool) *Logger {
```

Comment:
I was even wondering if we should drop `useJSON` to simplify this even more. In the future, a lot more tests will need a Logger, so this will be called often.

I was even wondering if JSON logs are the desired behaviour or if plain logs are easier to read. Thoughts?

Reply:
I suspect plain text is easier for console output for unit tests.  It may be easier to read within unit test output.

Reply:
Changed in 5b7a64eab6 to always use plain formatter.